builders/xcm/xcm-sdk.md

```diff
@@ -0,0 +1,678 @@
+---
+title: XCM SDK
+description: Use the Moonbeam XCM SDK to easily deposit and withdraw cross chain assets to Moonbeam from Polkadot and other parachains in the ecosystem.
+---
+
+# Using the Moonbeam XCM SDK
+
+![XCM SDK Banner](/images/builders/xcm/sdk/xcm-sdk-banner.png)
+
+## Introduction {: #introduction }
+
+The Moonbeam XCM SDK enables developers to easily deposit and withdraw assets to Moonbeam/Moonriver from the relay chain and other parachains in the Polkadot/Kusama ecosystem. With the SDK, you don't need to worry about determining the multilocation of the origin or destination assets or which extrinsics are used on which networks to send XCM transfers. To deposit or withdraw assets, you simply define the asset and origin chain you want to deposit from or withdraw back to, along with the sending account's signer, and the amount to send.
+
+The examples in this guide are shown on Moonbeam, but can be adapted to be used on Moonriver or Moonbase Alpha.
+
+Please note that the SDK is currently in heavy development and is not production ready.
```

Comment:
Not sure if we need this anymore. I was planning to remove this warning from the readme after the docs are live.